app/src/org/commcare/android/tasks/FormRecordCleanupTask.java

```diff
@@ -71,8 +71,10 @@ public FormRecordCleanupTask(Context context, CommCarePlatform platform, int tas
     protected Integer doTaskBackground(Void... params) {
         SqlStorage<FormRecord> storage = CommCareApplication._().getUserStorage(FormRecord.class);
 
-        Vector<Integer> recordsToRemove = storage.getIDsForValues(new String[] { FormRecord.META_STATUS}, new String[] { FormRecord.STATUS_SAVED });
-        int oldrecords = recordsToRemove.size();
+        Vector<Integer> recordsToRemove = storage.getIDsForValues(
+                new String[]{FormRecord.META_STATUS},
+                new String[]{FormRecord.STATUS_SAVED});

```

Comment:
@ctsims I had asked that in the PR comment above because I wasn't sure, but left it because no one said anything. Will change.